### ISSUE TITLE
Nested form - cells disappear

### ISSUE THREAD

**Issue by tallot13** [xmartlabs/Eureka#2099]
Created custom row like pushRow with value type Eureka.Form for nested forms.
for example created nested form with TextAreaRow 
after submission I change the value, go back and go to the form again, the edit field disappears, what could it be? disappears for various reasons if row.title is not set, or there was no value, but tap on the field, does anyone know how to get around this and why is this happening? the table cell itself has a height and a frame different from 0, but it is not in the hierarchy. 
isDisabled = false and isHidden = false

<img src="https://user-images.githubusercontent.com/48832276/95567252-57551f80-0a2b-11eb-800c-03933b486f18.png" width="200" height="400">
<img src="https://user-images.githubusercontent.com/48832276/95567260-5ae8a680-0a2b-11eb-830b-a83cee2f9c7e.png" width="200" height="400">
<img width="276" src="https://user-images.githubusercontent.com/48832276/95567908-4062fd00-0a2c-11eb-8cc5-2e551df3f0dd.png">

row class

    final class FormRow: Row<FormContentCell>, PresenterRowType, RowType {

cell class

    class FormContentCell: Cell<Form>, CellType {

use case nested

    form +++ Section()
            <<< FormRow() {
                let form = Form()
                $0.title = "title"
                $0.value = form
                $0.displayValueFor = { (value) in
                    return form.rowBy(tag: "title")?.baseValue as? String
                }
            }

nested form view controller

    class NestedFormViewController: FormViewController, TypedRowControllerType {
    
    public var row: RowOf<Form>!

    var onDismissCallback: ((UIViewController) -> Void)?
    
    required public init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
    }

    public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
        super.init(nibName: nil, bundle: nil)
    }

    convenience public init(_ callback: ((UIViewController) -> ())?){
        self.init(nibName: nil, bundle: nil)
        onDismissCallback = callback
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        form = row.value!
    }
    
    override func viewWillDisappear(_ animated: Bool) {
        super.viewWillDisappear(animated)
        row.baseCell.formViewController()?.tableView.reloadData()
        onDismissCallback?(self)
    }

**Comment by mats-claassen** [comment]
I have never tried nested forms. Are you not seeing any errors in the console?

**Comment by tallot13** [comment]
no errors.. tableview delegates called for every missed row and returned baseCell, if instead of baseCell return UITableViewCell(), then the cell will be empty, not missed

**Comment by mats-claassen** [comment]
So are you overriding the Tableview delegate methodds from Eureka? You should not need to do that.

BTW: Instead of `row.baseCell.formViewController()?.tableView.reloadData()` just call `tableView.reloadData()`

**Comment by tallot13** [comment]
no, not overriding, breakpoint in source code of Eureka FormViewController, upper all code nested form view controller
if added this bad code in NestedFormViewController, works fine. But how fix without this..
    
    override func viewDidAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        DispatchQueue.main.asyncAfter(deadline: .now() + 0.01) {
            self.tableView.reloadData()
        }
    }

**Comment by tallot13** [comment]
or like this works too
    
    override func viewDidLoad() {
        super.viewDidLoad()
        form = row.value!
        DispatchQueue.main.async {
            self.tableView.reloadData()
        }
    }
